ci-operator/step-registry/mtr/deploy-windup/mtr-deploy-windup-commands.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+# Deploy windup
+echo "Deploying Windup"
+oc apply -f - <<EOF
+apiVersion: windup.jboss.org/v1
+kind: Windup
+metadata:
+    name: mtr
+    namespace: $WINDUP_NAMESPACE
+    labels:
+      application: mtr
+spec:
+    volumeCapacity: "$WINDUP_VOLUME_CAP"
+EOF
+
+# Wait 5 minutes for Windup to fully deploy
+echo "Waiting 5 minutes for Windup to finish deploying"
```

Comment:
is there a command that we can use to proof that windup has been installed?

Then just move on once the condition is true or status says "ready". Adding a wait here may cause issue on a very slow network for some reason. Also if its up in 1 minute we spend an additional 4 minutes waiting than is needed.

Reply:
Not that I am aware of, but I can certainly take a look!

Reply:
Resolved in commit [4602739](https://github.com/openshift/release/pull/36181/commits/4602739b55370b75da05c3fc8352787545510f44)